operations/alloy-mixin/alerts/opentelemetry.libsonnet

```diff
@@ -1,7 +1,7 @@
 local alert = import './utils/alert.jsonnet';
 
 {
-  newOpenTelemetryAlertsGroup(enableK8sCluster=true):
+  newOpenTelemetryAlertsGroup(enableK8sCluster=true, percentThreshold='5', forT='5m'):
```

Comment:
I just took a look how this pans out in practice. There are a lot of temporary issues that recover within just a few minutes, so being notified of them after 5min by default may be a bit on a high end. I would propose changing `forT=15m` then when we get notified, we know that there is a good chance the issue persists and should be looked into.